server/src/main/java/com/google/finapp/FinAppService.java

```diff
@@ -58,28 +59,38 @@ public void createCustomer(
     responseObserver.onCompleted();
   }
 
+  private BigDecimal getNonNegativeBigDecimal(String value) throws StatusException {
```

Comment:
nit: I'd suggest moving this private method towards the end of the file so that it's consistent with the current ordering of the members.

Reply:
done